sdk/search/Azure.Search/src/Generated/Models/AutocompleteResult.cs

```diff
@@ -15,6 +15,6 @@ public partial class AutocompleteResult
         /// <summary> A value indicating the percentage of the index that was considered by the autocomplete request, or null if minimumCoverage was not specified in the request. </summary>
         public double? Coverage { get; internal set; }
         /// <summary> The list of returned Autocompleted items. </summary>
-        public ICollection<AutocompleteItem> Results { get; internal set; }
+        public IList<AutocompleteItem> Results { get; internal set; }
```

Comment:
So this change doesn't include the check to make this `IReadOnlyList<T>` then? Working on it, or should I rebase on this and continue the effort?

Reply:
Nope, this change fixed a low hanging fruit of blindly making everything `IList` work to make some things `IReadOnlyLists` is appreciated.